src/hooks/useGetIsTransferCompleted.ts

```diff
@@ -240,6 +243,24 @@ export default function useGetIsTransferCompleted(
             setIsLoading(false);
           }
         })();
+      } else if (targetChain === CHAIN_ID_SUI) {
+        setIsLoading(true);
```

Comment:
Is it possible there's a `setIsLoading(false)` missing in case of an error? I noticed it's missing for other chains too, so it'd be better to leave it for another PR/issue discussion.

Reply:
This is covered by line 261 (and matches the other functions). The error is caught and buried, then if the effect isn't cancelled loading will be set to false.